src/plugins/kibana_react/public/util/react_mount.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nProvider } from '@kbn/i18n/react';
+import { MountPoint } from 'kibana/public';
+
+/**
+ * Mount converter for react components.
+ *
+ * @param component to get a mount for
+ */
+export const reactMount = (component: React.ReactNode): MountPoint => {
+  const mount = (element: HTMLElement) => {
+    ReactDOM.render(<I18nProvider>{component}</I18nProvider>, element);
+    return () => ReactDOM.unmountComponentAtNode(element);
+  };
+  // used for proper snapshot serialization
+  mount.__reactMount__ = component;
+  return mount;
```

Comment:
the `__reactMount__ ` part is only using for tests and snapshots. We could remove it for production build, as it's private 'api'. Did not found any exemple of production code removal in kibana though, even if I know that's totally doable with webpack and env conditions. WDYT ?

Reply:
I believe `process.env.NODE_ENV === 'production'` should work.